tests/unittests/core/test_transformer.py

```diff
@@ -662,14 +705,20 @@ def test_no_requirement(self, space_each_type):
         assert tspace[0].type == 'real'
         assert tspace[1].type == 'categorical'
         assert tspace[2].type == 'integer'
-        assert str(tspace) == str(space_each_type)
+        assert (str(tspace) ==
+                "Space([Precision(Real(name=yolo, prior={norm: (0.9,), {}}, shape=(3, 2), default value=None)),\n"  # noqa
```

Comment:
I think `Precision.repr_format` should be overriden so that it prints `Precision({precision value}, Real(...))` otherwise we loose the information.